### ISSUE TITLE
Can't restore Thunderbird to its position

### ISSUE THREAD

**Issue by nlpsuge** [nlpsuge/gnome-shell-extension-another-window-session-manager#74]
From E.G.O:

> the latest version (36) wont restore thunderbird to its position no matter what i do, i just cant work it out. every other app is working. i reverted to version 34, which i had on a backup usb stick, no problems with that version.

Yes. I have tested on Fedora 37 with Gome 43.3. 35 and 36 both have this issue. 34 can restore the position.

**Comment by g1bber1sh** [comment]
No progress on this one?

**Comment by nlpsuge** [comment]
> No progress on this one?

Yes. Sorry about that. Really busy recently (for work and life).
